designer/client/src/components/graph/node-modal/fragment-input-definition/settings/variants/fields/UserDefinedListInput.tsx

```diff
@@ -47,24 +97,68 @@ export const UserDefinedListInput = ({ fixedValuesList, path, onChange, variable
             return true;
         },
     };
+
+    const processProperties = useSelector(getProcessProperties);
+    const scenarioName = useSelector(getProcessId);
+    const { processingType } = useSelector(getProcessToDisplay);
+    const temporaryItemName: FieldName = `$param.${name}.$fixedValuesListTemporaryItem`;
+
+    const validateTemporaryListItem = useMemo(() => {
+        return debounce(async (test: string) => {
+            const genericValidationRequest: GenericValidationRequest = {
+                parameters: [
+                    {
+                        name: temporaryItemName,
+                        typ: temporaryListItemTyp,
+                        expression: { language: ExpressionLang.SpEL, expression: test },
+                    },
+                ],
+                processProperties,
+                variableTypes: {},
+                scenarioName,
+            };
+
+            const response = await HttpService.validateGenericActionParameters(processingType, genericValidationRequest);
+
+            if (response.status === 200) {
+                setTemporaryValueErrors(response.data.validationErrors);
+            }
+
+            setTemporaryValuesTyping(false);
+        }, 500);
+    }, [processProperties, processingType, scenarioName, temporaryItemName, temporaryListItemTyp]);
+
     return (
         <SettingRow>
             <SettingLabelStyled>{t("fragment.addListItem", "Add list item:")}</SettingLabelStyled>
             <EditableEditor
-                fieldName="addListItem"
+                validationLabelInfo={temporaryValuesTyping && "Typing..."}
```

Comment:
I don't know if `Typing` is meaningful for our user. Maybe something less technical like `Validating`?

Reply:
Ahhh, in my mind in was "typing" as in "typing a value" in programming type :D I'm ok with this now